doc/lua_api.txt

```diff
@@ -6753,17 +6753,18 @@ object you are working with still exists.
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
 * `hud_set_flags(flags)`: sets specified HUD flags of player.
     * `flags`: A table with the following fields set to boolean values
-        * hotbar
-        * healthbar
-        * crosshair
-        * wielditem
-        * breathbar
-        * minimap
-        * minimap_radar
+        * `hotbar`
+        * `healthbar`
+        * `crosshair`
+        * `wielditem`
+        * `breathbar`
+        * `minimap`: Modifies the client's permission to view the minimap.
+          The client may locally elect to not view the minimap.
+        * `minimap_radar`: is only usable when `minimap` is true
+        * `debug`: Allow showing basic debug info that might give a gameplay advantage.
```

Comment:
My primary and obvious complain is here: shouldn't this be renamed `basic_debug` so people do not confuse it with the `debug` privilege?

Reply:
Now that the debug priv does indeed override this, probably yes.

Reply:
Last commit does this.